src/report.ts

```diff
@@ -17,12 +17,12 @@ export const logReport = (
 		// No table matching found
 
 		p.log.info(
-			`${green(`Type need to be manually typed`)}:
-  ${yellow(report.typeCouldBeBetter.map((c) => `${c}`).join("\n  "))}`
+			`${green("Type need to be manually typed")}:
```

Comment:
Could we tweak this? 

I like a lot `'` as you can add variables inside. With " you always have to change things...

Reply:
(I mean the backtick)

Reply:
sonar lint complainant about that but i can disable this rule 
what do you prefer?